compress_string.cpp

```diff
@@ -45,17 +38,17 @@ string compress(const string &to_compress)
 
 int main(int argc, char *argv[])
 {
-	string to_compress;
+	std::string to_compress;
 	if (argc == 2) {
 		to_compress = argv[1];
-
+		std::cout << "let's compress some shizz\n";
```

Comment:
use of `std::endl` is considered a pessimization in some contexts.

http://en.cppreference.com/w/cpp/io/manip/endl